internal/execute/table/buffered_builder_test.go

```diff
@@ -11,142 +12,159 @@ import (
 func TestBufferedBuilder_AppendTable(t *testing.T) {
 	for _, tt := range []struct {
 		name    string
-		in      []static.Table
+		in      static.TableGroup
 		want    static.Table
 		wantErr string
 	}{
 		{
 			name: "OneTable",
-			in: []static.Table{{
-				"_measurement": static.StringKey("m0"),
-				"_field":       static.StringKey("f0"),
-				"t0":           static.StringKey("a"),
-				"_time":        static.Times("2020-01-01T00:00:00Z", 10, 20, 30, 40),
-				"_value":       static.Ints(5, 3, nil, 9, 2),
-			}},
+			in: static.TableGroup{
+				static.StringKey("_measurement", "m0"),
+				static.StringKey("_field", "f0"),
+				static.StringKey("t0", "a"),
+				static.Table{
+					static.Times("_time", "2020-01-01T00:00:00Z", 10, 20, 30, 40),
+					static.Ints("_value", 5, 3, nil, 9, 2),
+				},
+			},
 			want: static.Table{
-				"_measurement": static.StringKey("m0"),
-				"_field":       static.StringKey("f0"),
-				"t0":           static.StringKey("a"),
-				"_time":        static.Times("2020-01-01T00:00:00Z", 10, 20, 30, 40),
-				"_value":       static.Ints(5, 3, nil, 9, 2),
+				static.StringKey("_measurement", "m0"),
+				static.StringKey("_field", "f0"),
+				static.StringKey("t0", "a"),
+				static.Times("_time", "2020-01-01T00:00:00Z", 10, 20, 30, 40),
+				static.Ints("_value", 5, 3, nil, 9, 2),
 			},
 		},
 		{
 			name: "Empty",
-			in: []static.Table{{
-				"_measurement": static.StringKey("m0"),
-				"_field":       static.StringKey("f0"),
-				"t0":           static.StringKey("a"),
-				"_time":        static.Times(),
-				"_value":       static.Ints(),
-			}},
+			in: static.TableGroup{
+				static.StringKey("_measurement", "m0"),
+				static.StringKey("_field", "f0"),
+				static.StringKey("t0", "a"),
+				static.Table{
+					static.Times("_time"),
+					static.Ints("_value"),
+				},
+			},
 			want: static.Table{
-				"_measurement": static.StringKey("m0"),
-				"_field":       static.StringKey("f0"),
-				"t0":           static.StringKey("a"),
-				"_time":        static.Times(),
-				"_value":       static.Ints(),
+				static.StringKey("_measurement", "m0"),
+				static.StringKey("_field", "f0"),
+				static.StringKey("t0", "a"),
+				static.Times("_time"),
+				static.Ints("_value"),
 			},
 		},
 		{
 			name: "TwoTables",
-			in: []static.Table{{
-				"_measurement": static.StringKey("m0"),
-				"_field":       static.StringKey("f0"),
-				"_time":        static.Times("2020-01-01T00:00:00Z", 10, 20, 30),
-				"_value":       static.Ints(4, 8, 2, 7),
-			}, {
-				"_measurement": static.StringKey("m0"),
-				"_field":       static.StringKey("f0"),
-				"_time":        static.Times("2020-01-01T00:00:40Z", 10, 20),
-				"_value":       static.Ints(3, 1, 9),
-			}},
+			in: static.TableGroup{
+				static.StringKey("_measurement", "m0"),
+				static.StringKey("_field", "f0"),
+				static.Table{
+					static.Times("_time", "2020-01-01T00:00:00Z", 10, 20, 30),
```

Comment:
So a time can be expressed either as an RFC339 string or an integer?

Reply:
Ok, I just read the doc comment for `Times`. The integers are offsets from the string times. I guess if an integer value appears first, it's an offset from the epoch.